Reduce `body_min_width` in basic theme

palewire opened the pull request:
The current 450px setting is too large for many small device width. It causes x-overflows that make the page appear broken and introduces difficulty reading on many phones.

palewire committed:
Reduce `body_min_width` in basic theme

tk0miya commented on sphinx/themes/basic/theme.conf:
```diff
@@ -7,7 +7,7 @@ sidebars = localtoc.html, relations.html, sourcelink.html, searchbox.html
 nosidebar = false
 sidebarwidth = 230
-body_min_width = 450
+body_min_width = 300
```
Is any reason to choose this value?

I just searched the screensizes of well-used mobile devices. Some reports says 360px is smallest size at present.
https://www.browserstack.com/guide/ideal-screen-sizes-for-responsive-design
https://mediag.com/blog/popular-screen-resolutions-designing-for-all/

palewire replied:
I picked it because it solved the bug. Another value could work fine if it also resolve the issue.

tk0miya committed:
Merge branch '5.x' into patch-1

tk0miya committed:
html theme: Change the default body_min_width to 360px

Some reports says 360px is smallest size at present.

refs:

- https://www.browserstack.com/guide/ideal-screen-sizes-for-responsive-design
- https://mediag.com/blog/popular-screen-resolutions-designing-for-all/

tk0miya committed:
Update CHANGES for PR #10197
